frontend/src/screens/dashboard/docsqa/main/partials/index.tsx

```diff
@@ -0,0 +1,8 @@
+import Option from './option'
+import ApplicationModal from './ApplicationModal'
+import Form from './Form'
+import NoAnswer from './NoAnswer'
+import ErrorAnswer from './ErrorAnswer'
+import Answer from './Answer'
+
+export { Option, ApplicationModal, Form, NoAnswer, ErrorAnswer, Answer }
```

Comment:
Lets avoid maintaining an `index.tsx`, I'd rather suggest using `index.tsx` in main screen pages where all the components are imported instead.

Reply:
I was thinking like in /screens we will have one main index.tsx which will contain path to pages only and inside page folder and inside that partials we can have one index.tsx more for the component path we can remove it as well it was optional.